.github/workflows/push.yml

```diff
@@ -0,0 +1,37 @@
+name: 'Continuous Integration'
+on:
+  push:
+    branches-ignore:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+jobs:
+  unit-tests:
+    runs-on: ubuntu-16.04
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v2-beta
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: 'unit tests'
+      run: |
+        npm install -g npm
+        npm install
+        npm run travis
+  npm-publish:
```

Comment:
I haven't tested this step yet since it's only set to run on pushes to `master`.

Reply:
We might want to replace this whole section with https://github.com/marketplace/actions/action-for-semantic-release